src/main/java/com/develhope/spring/vehicles/controller/VehicleController.java

```diff
@@ -1,21 +1,23 @@
 package com.develhope.spring.vehicles.controller;
 
 import com.develhope.spring.vehicles.entity.Vehicle;
-import com.develhope.spring.vehicles.service.VehicleServices;
+import com.develhope.spring.vehicles.service.VehicleService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/Vehicles")
+@RequestMapping("/vehicles")
 public class VehicleController {
     @Autowired
-    private VehicleServices vehicleServices;
+    private VehicleService vehicleServices;
 
     @PostMapping
-    public Vehicle addVehicle(@RequestBody Vehicle vehicle){
-        return vehicleServices.insertVehicle(vehicle);
+    public ResponseEntity<Vehicle> addVehicle(@RequestBody Vehicle vehicle){
+        Vehicle vehicleAdded = vehicleServices.insertVehicle(vehicle);
```

Comment:
Si potrebbe direttamente chiamare il metodo in return

Reply:
va be si, ma non è che non funziona se me lo salvo in una variabile, anzi.. mi sembra proprio più pulito il codice, io lo lascerei così

Reply:
Nono funzionare funziona, diciamo che dovrebbe essere un po' più leggero. Però va bene ;)